ocaml/xcp-rrdd/bin/rrdd/xcp_rrdd.ml

```diff
@@ -390,6 +403,18 @@ let dss_hostload xc domains =
         ~value:(Rrd.VT_Float load_per_cpu) ~min:0.0 ~ty:Rrd.Gauge ~default:true
         ()
     )
+  ; ( Rrd.Host
+    , Ds.ds_make ~name:"running_vcpus" ~units:"(fraction)"
```

Comment:
From the comment, this is a fraction. But from the calculation, this seems a count number?

Reply:
Fixed